src/export/export.js

```diff
@@ -121,6 +121,20 @@ module.exports = (router) => {
         return Promise.all(newComponents);
       };
 
+      //Replace duplicates from form.components if they form a group
+      const handleGroupComponents = (components) => {
+          let name;
+          return components.filter(component => {
+              if (!name && component.name) {
```

Comment:
This is incomplete. This would not be able to handle multiple components with duplicates.